docs/source/contributing_to_dallinger.rst

```diff
@@ -1,19 +1,50 @@
 Releasing a new version of Dallinger
 ====================================
 
-1. After you've merged the changes you want into `master`, start a new branch on which to run the version upgrade and update the CHANGELOG if that hasn't
+1. After you've merged the changes you want into `master`, start a new branch on
+which to run the release version upgrade and update the CHANGELOG if that hasn't
 been done as part of feature branch work.
 
 We’re using semantic versioning, so there are three parts to the version number. when making a release you need to decide which parts should get bumped, which determines what command you give to `bumpversion`. `major` is for breaking changes, `minor` for features, `patch` for bug fixes.
 
-Example:
-Running `bumpversion patch`, which will change every mention of the current version in the codebase and increase it by `0.0.1`.
+Under normal circumstances, the ``master`` branch will have a version number
+reflecting a `minor` increment over the previous release, plus a `a1` suffix
+indicating that it includes unreleased ("alpha") changes. If the release you're
+making is indeed a `minor` release, then simply run ``bumpversion release``.
+This will remove the `a1` suffix, leaving the correct minor version increment.
+
+If you have breaking changes and need to switch to a `major` release, then you
+will need to first make that change, commit, and then issue the release increment
+command::
+
+    $ bumpversion major
+    $ git commit -a -m "Switch to major release"
+    $ bumpverion release
+    $ git commit -a -m "Update versions for release"
+
+In instead you need to switch to a `patch` release, you will instead need to
```

Comment:
I don't think `bumpversion patch` will work, because in the case of a patch release, you've already overshot the version you actually want. For example, master is currently on 7.8.0a1, and a patch release should actually go back to 7.7.1, or similar, but `bumpversion patch` results in 7.8.1a1. Did you see something different?

```
-    version="7.8.0a1",
+    version="7.8.1a1",
```

Reply:
Fair enough! I was focusing on the tree and lost track of the forest.